packages/flutter_tools/lib/src/commands/emulators.dart

```diff
@@ -65,6 +69,18 @@ class EmulatorsCommand extends FlutterCommand {
     }
   }
 
+  Future<Null> _createEmulator(String name) async {
+    final CreateEmulatorResult createResult =
+        await emulatorManager.createEmulator(name);
+
+    if (createResult.success) {
+      printStatus("Emulator '$name' created successfully.");
+    } else {
+      printStatus(createResult.error);
+      printStatus("Failed to create emulator '$name'.");
```

Comment:
We should print some text here about where they can find more info about creating emulator images (referencing flutter.io, or Android / Android Studio docs).

Reply:
How's this? Seems a bit weird sending people to Android Studio or avdmanager docs but since the errors come from them I'm not sure if we can provide the info ourselves.

<img width="523" alt="screen shot 2018-06-11 at 6 00 30 pm" src="https://user-images.githubusercontent.com/1078012/41245950-6df61d1c-6da1-11e8-8f4d-df2be22c9d7d.png">

Reply:
This looks great for error text. We'll probably also want something brief for `flutter emulators -h` help text as well - perhaps pointing to a flutter.io page?

@mit-mit, @mjohnsullivan - we may want to consider a brief landing page on flutter.io describing how to create android emulators, or at least just routing people to places to find for info about it.